### PR TITLE
enabling create app in current directory

### LINKED ISSUE #110: Create project in current directory
So I want to upgrade my current git project with preact. This app is just html and JS. So how can I create a new project in the current directory instead of creating a new folder? Thanks

I tried this:

`preact create lcac --type root --dest . `

### PR DESCRIPTION
**Pull request by prateekbh**
Fixes #110 
by specifying --force this will allow you to create project in current directory.

@developit just curious, the discussion emphasize upon the usage of `--force`?
What if we do it without this? Whats the harm with that?

### CODE REVIEW

File: src/commands/create.js [hunk 1]
```diff
@@ -85,8 +87,25 @@ export default asyncCommand({
 		}
 		catch (err) {}
 
+		if (exists && argv.force) {
+			const question = {
+        type: 'confirm',
+        name: 'enableForce',
+        message: `You are using '--force'. Do you wish to continue?`,
+        default: true,
```
Comment (prateekbh) [inline src/commands/create.js]: default should be false

File: src/commands/create.js [hunk 2]
```diff
@@ -85,8 +87,25 @@ export default asyncCommand({
...
+			if (forceInit.enableForce) {
+				process.stdout.write('Initializing project in the current directory...\n');
+			} else {
+				throw Error('Cannot initialize the project in the current directory');
+			}
+		}
+
 		if (exists && !argv.force) {
-			throw Error('Directory already exists.');
+			throw Error('Cannot intialize in the current directory, please specify a different destination.');
```
Comment (prateekbh) [inline src/commands/create.js]: can we write messages using chalk?

File: src/commands/create.js (hunk 1 above)
Comment (reznord) [inline src/commands/create.js]: Okay, will change it once I reach home.

_Sent from my OnePlus ONEPLUS A3003 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

File: src/commands/create.js (hunk 2 above)
Comment (reznord) [inline src/commands/create.js]: Oh sure, not a problem at all. Red color?

_Sent from my OnePlus ONEPLUS A3003 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

File: src/commands/create.js [hunk 3]
```diff
@@ -81,16 +87,37 @@ export default asyncCommand({
 		}
 		catch (err) {}
 
-		if (exists) {
-			throw Error('Directory already exists.');
+		if (exists && argv.force) {
+			const question = {
+        type: 'confirm',
```
Comment (developit) [inline src/commands/create.js]: whitespace <3

File: src/commands/create.js [hunk 4]
```diff
@@ -81,16 +87,37 @@ export default asyncCommand({
...
+				message: `You are using '--force'. Do you wish to continue?`,
+				default: false,
+			};
+
+			let forceInit = await inquirer.prompt(question);
+
+			if (forceInit.enableForce) {
+				process.stdout.write('Initializing project in the current directory...\n');
+			} else {
+				process.stdout.write(`${chalk.red('Error:')} Cannot initialize in the current directory`);
```
Comment (rkostrzewski) [inline src/commands/create.js]: Should be `process.stderr.write`. BTW why not color whole message to red?

File: src/commands/create.js [hunk 5]
```diff
@@ -81,16 +87,37 @@ export default asyncCommand({
...
+				process.exit(1);
+			}
+		}
+
+		if (exists && !argv.force) {
+			process.stdout.write(`${chalk.red('Error:')} Cannot initialize in the current directory, please specify a different destination`);
```
Comment (rkostrzewski) [inline src/commands/create.js]: Should be `process.stderr.write`

File: src/commands/create.js [hunk 6]
```diff
@@ -81,16 +87,37 @@ export default asyncCommand({
...
-			throw Error('Directory already exists.');
+		if (exists && argv.force) {
+			const question = {
+				type: 'confirm',
+				name: 'enableForce',
+				message: `You are using '--force'. Do you wish to continue?`,
+				default: false,
+			};
+
+			let forceInit = await inquirer.prompt(question);
```
Comment (rkostrzewski) [inline src/commands/create.js]: Hint: `let { enableForce } = `, also `enableForce`:
enableForce

**Comment by reznord** [comment]
Let the tests run !! :P Merging this one 😄

**Pull request merged**